pkg/handler/crud.go

```diff
@@ -142,6 +186,7 @@ func (c *crud) CreateFlag(params flag.CreateFlagParams) middleware.Responder {
 	resp.SetPayload(payload)
 
 	entity.SaveFlagSnapshot(getDB(), f.ID, getSubjectFromRequest(params.HTTPRequest))
```

Comment:
should this be part of the transaction somehow? just mimicking other endpoints right now but maybe something for followup

Reply:
yes, tracked here https://github.com/checkr/flagr/issues/355